<launch>

<!-- start roomba driver node -->
	<node pkg="roomrider" type="roomba_driver" output="screen" name="roomrider" >
	</node>

<!-- start laser driver node -->
	<node pkg="roomrider" type="sick300_driver" output="screen" name="sick300" >
	</node>

</launch>